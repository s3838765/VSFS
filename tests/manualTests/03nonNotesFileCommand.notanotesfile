NOTES V1.0
@invalidfilesystemfile
 This is an invalid file,
 in an invalid file system.
 This should never work!
