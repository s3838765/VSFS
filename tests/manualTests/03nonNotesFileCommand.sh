#! /usr/bin/sh
../../VSFS list 03nonNotesFile.notanotesfile 1> 03nonNotesFile.out 2>&1
