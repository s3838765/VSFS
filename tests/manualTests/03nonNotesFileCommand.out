The specified file system does not exist. Terminating program.
